task8/TutterApp/src/main/webapp/index.html

```diff
@@ -10,65 +10,96 @@
         <link href="http://fonts.googleapis.com/css?family=Reenie+Beanie" rel="stylesheet">
         <link href="http://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
         <script src="https://kit.fontawesome.com/9b7f939454.js" crossorigin="anonymous"></script>
-        <script src="resources/js/script.js"></script>
+        <script src="resources/js/model.js"></script>
+        <script src="resources/js/view.js"></script>
+        <script src="resources/js/controller.js"></script>
     </head>
 
     <body>
     <header>
-        <div class="logo">
+        <div id="logo" class="logo">
```

Comment:
You don't need add class `logo` to `img` and `p` tags. One class for div is enough.
In CSS you can use selectors `.logo img` and `.logo p `